packages/migrate/migrations/svelte-5/index.js

```diff
@@ -190,10 +190,8 @@ export async function migrate() {
 		"install the updated dependencies ('npm i' / 'pnpm i' / etc) " +
 			'(note that there may be peer dependency issues when not all your libraries officially support Svelte 5 yet. In this case try installing with the --force option)',
 		use_git && cyan('git commit -m "migration to Svelte 5"'),
-		'Review the breaking changes at https://svelte-5-preview.vercel.app/docs/breaking-changes'
-		// replace with this once it's live:
-		// 'Review the migration guide at https://svelte.dev/docs/svelte/v5-migration-guide',
-		// 'Read the updated docs at https://svelte.dev/docs/svelte'
+		'Review the migration guide at https://svelte.dev/docs/svelte/v5-migration-guide',
+		'Read the updated docs at https://svelte.dev/docs/svelte'
```

Comment:
personally I'd remove the second sentence eventhough it was here commented out. I think the first one's probably enough
```suggestion
		'Review the migration guide at https://svelte.dev/docs/svelte/v5-migration-guide'
```

Reply:
agreed, it's a bit redundant now that they share the same site